### PR TITLE
[bluetooth.govee] null annotations

### PR DESCRIPTION
**Pull request by lsiepel**
Signed-off-by: Leo Siepel [leosiepel@gmail.com](mailto:leosiepel@gmail.com)

null annotations
checkstyle fixes

Part of multiple PR's for bluetooth bundles. Can be merged independent.

### CODE REVIEW

**Comment by lsiepel** [comment]
ThingTypeTableGenerator.java has two compile warnings left. If they need to get fixed, i could use some suggestions.

File: bundles/org.openhab.binding.bluetooth.govee/src/main/java/org/openhab/binding/bluetooth/govee/internal/command/hygrometer/GetOrSetHumWarningCommand.java [hunk 1]
```diff
@@ -65,14 +65,20 @@ private static short convertQuantity(QuantityType<Dimensionless> quantity) {
 
     @Override
     protected byte @Nullable [] getData() {
-        if (settings == null) {
+        WarningSettingsDTO<Dimensionless> localSettins = settings;
+        if (localSettins == null || localSettins.min == null || localSettins.max == null) {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.bluetooth.govee/src/main/java/org/openhab/binding/bluetooth/govee/internal/command/hygrometer/GetOrSetHumWarningCommand.java:69]: Min and max are checked just after.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.bluetooth.govee/src/main/java/org/openhab/binding/bluetooth/govee/internal/command/hygrometer/GetOrSetHumWarningCommand.java (hunk 1 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.bluetooth.govee/src/main/java/org/openhab/binding/bluetooth/govee/internal/command/hygrometer/GetOrSetHumWarningCommand.java:72]: Fixed.

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**